### DISCUSSION TITLE
Debug - log file location

### DISCUSSION THREAD

**Discussion by Boss-man** [karakeep-app/karakeep#746] [General]
hi there

I have Hoarder installed in LXC on Proxmox by the Helper-script.

**Where do I find log files, to see whats going on when crawling a site ?**
For instance this site give me error: https://www.digitec.ch/en/page/synology-nas-upgrade-25-gigabits-with-a-usb-lan-adapter-27363

**How do I connect to the remote chrome ?**
Should I edit the .env file (/opt/hoarder/.env) and change BROWSER_WEB_URL="http://127.0.0.1:9222", to the LXC IP = http://192.168.1.x:9222  ?

I did try to look at the documentation, but couldn't find anything on looking at logs or remote debugging.

**Comment by MohamedBassem** [comment]
hmmm, I think this is an oversight from us that we're not persisting the logs to a directory in the LXC installation. I'll file an issue to fix that.

Can you run `journalctl -u  hoarder` and see if it shows any logs?

**Comment by Boss-man** [comment]
it just returns "-- No entries --"

update:
If I just do a normal journalctl I do get lots of log entries from meilisearch and pnpm.

**Comment by 2001db8** [comment]
Stumbled across this too trying to figure out why OpenAI did not work (Error message in the Ui is not helping at all... ;) )

@Boss-man If you are looking for anything special you can use `journalctl -f -g "keyword"` and journalctl will search for `keyword` and update the output like with `tail -f`.
